components/lib/crash/src/main/java/mozilla/components/lib/crash/CrashReporter.kt

```diff
@@ -130,7 +130,7 @@ class CrashReporter(
 
         return scope.launch {
             services.forEach {
-                it.report(throwable)
+                it.report(throwable, crashBreadcrumbs.toSortedArrayList())
             }
```

Comment:
Since a manual report won't end the app/process: Wouldn't we add and report the same breadcrumbs multiple times if we submit multiple exceptions?
Do we need to somehow clear our list of breadcrumbs after they have been recorded once?

Reply:
I was back and forth about this.  At the end I decided we shouldn't clear it.  The main reason is if there is a uncaught exception that happens quickly after a caught exception, the breadcrumbs will still help debug the issue.  

I think keeping them is better since we can't be sure which breadcrumb belongs to which report.

Reply:
I worry that duplicated breadcrumbs may be misleading and confusing when looking for causes? But then again I guess this can only happen if we are in the same process and reusing the same Sentry instance?
What if we call `clearBreadcrumbs()` on Sentry itself after we reported a crash? This way we can keep them ourselves and re-add them the next time we report something?

Reply:
I see what you mean.  Make sense.  I'll update.  Thanks,